cmake/onnxruntime_common.cmake

```diff
@@ -48,10 +48,14 @@ source_group(TREE ${REPO_ROOT} FILES ${onnxruntime_common_src})
 add_library(onnxruntime_common ${onnxruntime_common_src})
 
 if (onnxruntime_USE_MIMALLOC)
-    include(external/mimalloc.cmake)
-    list(APPEND onnxruntime_EXTERNAL_LIBRARIES mimalloc-static)
-    list(APPEND onnxruntime_EXTERNAL_DEPENDENCIES mimalloc-static)
-    target_link_libraries(onnxruntime_common mimalloc-static)
+    if(onnxruntime_USE_CUDA OR onnxruntime_USE_OPENVINO) 
```

Comment:
But we also have mkldnn/ngraph/...

Reply:
Is there a more flexible way to check for the cpu target?

Reply:
I'm not aware of any.

Reply:
Testing appears to be fine for ngraph/mkldnn so I'm going to leave the flag on for those two...it looks like the windows CI builds with mkldnn so at the very least that one is required. The original motivation for turning this off on CUDA/OpenVINO were strange crashes.